packages/dd-trace/test/appsec/index.fastify.plugin.spec.js

```diff
@@ -264,7 +264,11 @@ withVersions('fastify', 'fastify', version => {
   })
 
   describe('Suspicious request blocking - path parameters', () => {
-    let server, paramHookSpy, axios
+    // Skip Fastify v1 - preValidation hook is not supported
+    if (semver.lt(semver.coerce(version), '2.0.0')) {
```

Comment:
if we don't really support blocking before 2.0.0 we should consider appsec only support fastify starting from v2 and add that to the relevant doc